src/components/swimlanes/swimLanes.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  CardHeading,
+  Flex,
+  Heading,
+  SimpleGrid,
+  Spacer,
+  Text,
+  Link,
+} from "@nypl/design-system-react-components";
+import React from "react";
+
+const SwimLanes = ({ lanesWithNumItems }) => {
+  return (
+    <>
+      {lanesWithNumItems.map((lane, key) => (
+        <Box data-testid={lane.slug} key={lane.rank} mb="var(--nypl-space-l)">
+          <Flex alignItems="baseline">
+            <Heading id={`row-heading-${lane.slug}`} level="h2" size="heading3">
+              {lane.title}
+            </Heading>
+            <Spacer />
+            <Link
+              id={`row-see-more-${lane.slug}`}
+              type="standalone"
+              href={"https://digitalcollections.nypl.org/search/index?"}
+              aria-label="See more digitized materials"
+            >
+              See more
+            </Link>
+          </Flex>
+          {lane.collections && lane.collections.length > 0 && (
+            <SimpleGrid columns={4} id={`grid-${key}`}>
+              {lane.collections.map((collection, index) => (
+                <Card
+                  key={index}
+                  id={`card-${key}-${index}`}
+                  imageProps={{
+                    alt: collection.title,
+                    aspectRatio: "twoByOne",
+                    src: `https://iiif.nypl.org/iiif/2/${collection.image_id}/full/288,/0/default.jpg`,
+                  }}
+                >
+                  <CardHeading
+                    id={`row-card-heading-${key}-${index}`}
+                    level="h3"
+                    size="heading4"
+                    url={collection.url}
+                    style={{
+                      display: "-webkit-box",
+                      WebkitLineClamp: 3,
+                      WebkitBoxOrient: "vertical",
```

Comment:
I'm not familiar with these rules. Why do you need them?

Reply:
These styles are the only solution I could find that would create the ellipsis when there's text overflow past the third line

Reply:
Yes, makes sense. Briefly chatted with @bigfishdesign13 (Marty) about this. I was concerned that the webkit prefix would not be supported in Firefox and other browsers but it looks like it's well supported: https://caniuse.com/?search=webkit-line-clamp.

Can you try it without the webkit prefix, so just `lineClamp`, and see if that still displays correctly in Chrome and Firefox? 

One other concern is that the ellipsis is not blue or (it seems) part of the link. Not sure yet if that's a big issue or not.

Reply:
No go without the webkit prefix unfortunately. And yes I don't know if that's a problem

Reply:
I would suggest including both the standard and the `webkit` version of the two attributes.  This is still common for some CSS attributes.  In fact, I am suprised the packager does not take care of that for us as it does with some other CSS attributes.

Reply:
I don't think postcss was added or configured to add the prefixes. @7emansell that can be done at a later time with a specific ticket that has better details.